numba/cuda/tests/cudapy/test_fastmath.py

```diff
@@ -5,6 +5,7 @@
 
 
 class TestFastMathOption(CUDATestCase):
+    @skip_on_cudasim('fast divide not available in CUDASIM')
     def test_kernel(self):
```

Comment:
For a while I was wondering what this is checking that differs from what `test_divf` checks for below... Eventually I concluded that it tests that casting an integer to a `float32` then using it in a division results in a `div.approx` instruction being emitted - given how finicky casts and the type system can be, I think it's good to keep this check alongside `test_divf` - Could you add a comment here explaining that this is testing the cast of an int being used in the divide, to make it a bit easier for future readers to see the difference?

Alternatively - do you have a different view on the difference between this and `test_divf`? I reached my conclusion by trying to derive meaning from the code (thinking backwards a bit) so maybe there's something else going on here that I haven't followed.